src/node.cc

```diff
@@ -167,6 +171,10 @@ static const char* icu_data_dir = nullptr;
 // used by C++ modules as well
 bool no_deprecation = false;
 
+// Unhandled promises tracking
+Persistent<NativeWeakMap>* remainingURs;
+Persistent<Object>* first_ur_key;
```

Comment:
I don't understand 2 but `Persistent` is just a value type.  You assign it a `Local` through the `.Reset()` method.  Or is your question how to turn a `Persistent` into a `Local` again?